packages/components/src/Helpers.purs

```diff
@@ -117,3 +122,17 @@ getMinStartDate nextDelivery = do
           soonestStart = adjust soonestDuration (DateTime.date now)
           byNextIssue = max <$> soonestStart <*> nextDelivery
       pure $ byNextIssue <|> soonestStart
+
+matchesRegex :: String -> String -> Boolean
+matchesRegex regex str = isJust $ do
+  re <- hush $ Regex.regex regex mempty
+  Regex.match re str
+
+-- This same deterministic function is in Lettera's code
+editorialIdToUuid :: String -> UUID.UUID
+editorialIdToUuid editorialId =
+  UUID.genv5UUID ("https://hblmedia.fi/" <> editorialId) url_namespace
+
+-- Constant defined in the UUIDv5 standard
```

Comment:
Add a link as a reference to where this is defined

Reply:
Fixed